docs/v3.x/migration-guide/migration-guide-3.2.5-to-3.3.0.md

```diff
@@ -38,4 +40,22 @@ export const SETTINGS_BASE_URL = '/settings';
 export const STRAPI_UPDATE_NOTIF = true;
 ```
 
+**Notification**
+
+If you use the `strapi.notification` global to display notifications, you will be able to update them with the new api.
+
+_Before_
+
+```js
+strapi.notification.success('app.notification.success');
+```
+
+_After_
+
+```js
+strapi.notification.toggle(config);
+```
+
+Here is a link to the [new notification API](https://strapi.io/documentation/v3.x/plugin-development/frontend-development.html#api)
```

Comment:
Use relative links to the .md please so that we don't have to monitor hard coded links

Reply:
👍🏽

Reply:
Done @derrickmehaffy